developments_build/sql/yearly.sql

```diff
@@ -0,0 +1,131 @@
+/*
+DESCRIPTION:
+    Creates an unaggregated table as input for the spatial aggregates.
+    Includes completions split by year. 
+    
+INPUTS:
+    FINAL_devdb (
+        * job_number,
+        job_type,
+        date_permittd,
+        date_complete,
+        date_lastupdt,
+        classa_net,
+        boro,
+        bctcb2010,
+        bct2010,
+        nta2010,
+        ntaname2010,
+        puma2010
+    )
+
+    LOOKUP_fips(
+        boro,
+        fips_boro
+    )
+
+OUTPUTS:
+    YEARLY_devdb (
+        * job_number,
+        boro,
+        bctcb2010,
+        bct2010,
+        nta2010,
+        ntaname2010,
+        puma2010,
+        comp2010ap,
+        comp2010,
+        comp2011,
+        comp2012,
+        comp2013,
+        comp2014,
+        comp2015,
+        comp2016,
+        comp2017,
+        comp2018,
+        comp2019,
+        comp2020,
+        comp2020q2,
+        incmpfiled,
+        incmpprgrs,
+        incmprmtd,
+        incmpwtdrn,
+        inactive
+    )
+
+*/
+
+
+DROP TABLE IF EXISTS YEARLY_devdb;
+SELECT a.job_number,
+        a.boro,
+        a.bctcb2010,
+        b.fips_boro||RIGHT(a.bctcb2010, 10) as cenblock10,
+        a.bct2010,
+        b.fips_boro||RIGHT(a.bct2010, 6) as centract10,
+        a.nta2010,
+        a.ntaname2010,
+        a.puma2010,
+        CASE WHEN a.complete_year = '2010' AND a.date_complete > '2010-03-31'::date
```

Comment:
I see a sustainability issue with this because we'll need to manually add a new year every year.  Would it be better to create the classa_net summary table by complete_year and then add on the additional fields?  If not, we need to have this well documented

Reply:
this is adding additional fields too? I don't see the difference